File: PHP/StackArray.php
<?php
class Stack {
    function __contruct(){
        $this->arr = array();
    }

    function push($data){
        array_push($this->arr,$data);
    }

    function pop(){
        if (count($this->arr) === 0){
            echo "Stack Empty";
        }
        return array_pop($this->arr);
    }

    function peek(){
        if (count($this->arr) === 0){
            echo "Stack Empty";
        }
        return $this->arr[count($this->arr)-1];
    }

    function isEmpty(){
        if (count($this->arr) === 0){
            return true;
        }
        return false;
    }

    function size(){
        return count($this->arr);
    }

    function printList(){
        foreach($this->arr as $item){
            echo $item;
        }
    }
}

$stack = new Stack();
echo $stack->isEmpty();
	
$stack->push(1);
$stack->push(3);
$stack->push(5);
$stack->push(7);
$stack->push(9);

echo $stack->peek();
$stack->printList();
echo $stack->pop();
$stack->isEmpty();

?>

File: Python/StackArray.py
class Stack:
    def __init__(self):
        self.arr = []
    
    def push(self, data):
        self.arr.append(data)
    
    def pop(self):
        if self.arr is None:
            print("Stack Empty")
        return self.arr.pop()
    
    def peek(self):
        if self.arr is None:
            print("Stack Empty")
        return self.arr[len(self.arr)-1]

    def isEmpty(self):
        if self.arr is None:
            return True
        return False

    def size(self):
        return len(self.arr)

    def print_list(self):
        for i in self.arr:
            print(i)


stack_array = Stack()
stack_array.isEmpty()

stack_array.push(1)
stack_array.push(3)
stack_array.push(5)
stack_array.push(7)
stack_array.push(9)

stack_array.print_list()
print(stack_array.isEmpty())	
print(stack_array.peek())
print(stack_array.pop())
